homeassistant/helpers/entity.py

```diff
@@ -1664,8 +1664,7 @@ class ToggleEntity(
     _attr_state: None = None
 
     @property
-    @final
-    def state(self) -> Literal["on", "off"] | None:
+    def state(self) -> str | None:
```

Comment:
Removes the @final decorator and changes the return type annotation to allow subclass Entities to override

Reply:
I wonder if we should add another base class that entities that need to override the state should use. Most entities that inherit ToggleEntity should not override the state property.